pkg/datagatherer/k8s/fieldfilter_test.go

```diff
@@ -74,68 +82,44 @@ func TestSelect(t *testing.T) {
 					"destinationCACertificate": "destinationCaCert data",
 				},
 			},
+		}, RouteSelectedFields,
+		map[string]interface{}{
+			"apiVersion": "v1",
+			"kind":       "Route",
+			"metadata": map[string]interface{}{
+				"name": "example",
+				"annotations": map[string]interface{}{
+					// The "last-applied-configuration" isn't ignored in
+					// "Select". "Redact" removes it.
+					"kubectl.kubernetes.io/last-applied-configuration": "secret",
+				},
```

Comment:
As per the Jira ticket, I haven't included the Route labels. Should I add them? @tfadeyi

Reply:
Looks good! thank you @maelvls!  There's no need to add them to the routes at the moment :+1: